caravel/templates/listing_show.html

```diff
@@ -18,28 +18,26 @@
           {% else %}
             <div class="item">
           {% endif %}
-
-          {% if photo %}
             <img src="{{ photo | public_url }}">
-          {% else %}
-            <div class="large-image-placeholder">
-              {{ listing.body[:1].upper() }}
-            </div>
-          {% endif %}
-
-{#          Folloring div closes item active or item#}
           </div>
         {% set i = i + 1 %}
```

Comment:
Why len?

This exists so that the first div tag in the carosel is "item-active" and the rest are "item"